js/bundles/modules/parts/viz.js

```diff
@@ -1,5 +1,6 @@
 var DevExpress = require("./core");
 require("./data");
+require("./file_providers");
```

Comment:
The FileManager parts should not contain in the Viz bundle

Reply:
Done